server/routes/getting_started/helper.ts

```diff
@@ -8,8 +8,12 @@ import path from 'path';
 
 export const assetMapper = (tutorialId: string) => {
```

Comment:
We may want to consider refactor this code piece as it is
- not easy to maintain, the growing number of cases makes these magic strings 
  - proven to typos
  - not easy to be modified when naming and version changes along the way

we want to consider 
- use map or enum for all these file names and move these stuff to constant file.
```typescript
type TutorialId = 'otelLogs' | 'otelMetrics' | 'otelTraces' | 'nginx' | 'java' | 'python' | 'golang';

const assetMap: Record<TutorialId, string> = {
  otelLogs: 'otel-index-patterns-1.0.0-Logs.ndjson',
  otelMetrics: 'otel-index-patterns-1.0.0-Metrics.ndjson',
  otelTraces: 'otel-index-patterns-1.0.0-Traces.ndjson',
  nginx: 'nginx-1.0.0.ndjson',
  java: 'java-tutorial-1.0.0.ndjson',
  python: 'python-tutorial-1.0.0.ndjson',
  golang: 'golang-tutorial-1.0.0.ndjson',
};

export const assetMapper = (tutorialId: TutorialId): string => {
  return assetMap[tutorialId] || '';
};
```
- maybe use concatenations like `${component}-${indexPattens}-${version}-${signal}.ndjson`
- provides a default return?

Also similar question: are these files coming from integration? If so, do we need to maintain these files from within get started module? Can't we just fetch from somewhere?

Reply:
These are all currently manually created.

Reply:
Adjusted it to be
```
export const assetMapper = (tutorialId: TutorialId): string => {
  const component = COMPONENT_MAP[tutorialId] || 'default-component';
  const version = VERSION_MAP[tutorialId] || '1.0.0';
  const signal = SIGNAL_MAP[tutorialId] ? `-${SIGNAL_MAP[tutorialId]}` : '';

  return `${component}-${version}${signal}.ndjson`;
};
```
and moved others to constants.